vyper/codegen/function_definitions/utils.py

```diff
@@ -1,8 +1,16 @@
+from vyper.semantics.types.function import StateMutability
+
+
 def get_nonreentrant_lock(func_type):
     if not func_type.nonreentrant:
         return ["pass"], ["pass"]
 
     nkey = func_type.reentrancy_key_position.position
-    nonreentrant_pre = [["seq", ["assert", ["iszero", ["sload", nkey]]], ["sstore", nkey, 1]]]
-    nonreentrant_post = [["sstore", nkey, 0]]
-    return nonreentrant_pre, nonreentrant_post
+
+    if func_type.mutability == StateMutability.VIEW:
+        return [["assert", ["iszero", ["sload", nkey]]]], [["seq"]]
```

Comment:
why `(seq)` instead of `(pass)` like above?

Reply:
`seq` now allows 0 arguments -- 0 argument `seq` is equivalent to `pass`. i'm trying to use `seq` going forward instead of `pass` (and eventually remove `pass` entirely).